From boris.brezillon@free-electrons.com  Fri May  8 13:05:22 2015
From: Boris Brezillon <boris.brezillon@free-electrons.com>
Date: Thu, 7 May 2015 11:56:44 +0200
To: Luis Henriques <luis.henriques@canonical.com>
Subject: clk: at91: usb: fix determine_rate prototype
Cc: Tyler Baker <tyler.baker@linaro.org>, Greg Kroah-Hartman <gregkh@linuxfoundation.org>, "linux-kernel@vger.kernel.org" <linux-kernel@vger.kernel.org>, torvalds@linux-foundation.org, Andrew Morton <akpm@linux-foundation.org>, linux@roeck-us.net, shuah.kh@samsung.com, stable@vger.kernel.org, Mike Turquette <mturquette@linaro.org>
Message-ID: <20150507115644.1ef29d2e@bbrezillon>

From: Boris Brezillon <boris.brezillon@free-electrons.com>

Commit c67881fc890916206e723329e774391c6ed354ce is a backport of
0b67c43ce36a9964f1d5e3f973ee19eefd3f9f8f upstream commit, fixing a
bug on clk rate change propagation.
But in 4.0 ->determine_rate() prototype has changed, thus introducing
a prototype mismatch when applying it on 3.19.

Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/clk/at91/clk-usb.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/clk/at91/clk-usb.c
+++ b/drivers/clk/at91/clk-usb.c
@@ -58,8 +58,6 @@ static unsigned long at91sam9x5_clk_usb_
 
 static long at91sam9x5_clk_usb_determine_rate(struct clk_hw *hw,
 					      unsigned long rate,
-					      unsigned long min_rate,
-					      unsigned long max_rate,
 					      unsigned long *best_parent_rate,
 					      struct clk_hw **best_parent_hw)
 {
